src/status_im/utils/ethereum/macros.clj

```diff
@@ -21,10 +29,19 @@
   (mapv #(assoc-in % [:icon :source] (token->icon network %)) tokens))
 
 (defn network->icon [network]
-  (let [s (str "./resources/images/tokens/" (name network) "/0-native.png")]
+  (let [s     (str "./resources/images/tokens/" (name network) "/0-native.png")
```

Comment:
can you explain this change?

Reply:
Do you explain why we need to load messages only be request and not right away or what exactly?